examples/extend_web_ui/extend.py

```diff
@@ -9,9 +9,16 @@
 import os
 from html import escape
 from time import time
+from warnings import warn
 
 from flask import Blueprint, jsonify, make_response, render_template
 
+warn(
+    "This UI example is deprecated, please downgrade to Locust 2.21.0 or use --legacy-ui flag ; version=2.22.0",
```

Comment:
Since this example would continue to work with the `--legacy-ui` flag, I think it would make more sense to suggest this as a solution rather than downgrading right?

Reply:
"This UI example is deprecated, make sure to use --legacy-ui flag ; version=2.22.0" ?

Reply:
I've added a deprecation warning log when --legacy-ui is active and will probably remove it in 2.24, so lets not worry too much about it :)